microsoft-edge/progressive-web-apps-chromium/get-started.md

```diff
@@ -115,21 +115,20 @@ Service workers are background tasks that intercept network requests from your w
     
 Service workers are defined in a special JavaScript file.  For more information, navigate to [Using Service Workers][MDNUsingServiceWorkers] and [Service Worker API][MDNServiceWorkerApi].  
 
-To build a service worker in your project, use the **Cache-first network** service worker recipe from [PWA Builder][PwaBuilderServiceWorker].  
+To build a service worker in your project, use the **Cache-first network** service worker recipe.  
 
-1.  Navigate to [pwabuilder.com/serviceworker][PwaBuilderServiceWorker], select the **Cache-first network** service worker, and select the **Download** button.  The downloaded file contains the following files:
+1. Copy the source files [pwabuilder-sw-register.js][PwaBuilderServiceWorkerRegisterJS] and [pwabuilder-sw.js][PwaBuilderServiceWorkerJS] to the `public` folder in your web app project.
     
     *   `pwabuilder-sw-register.js`  
     *   `pwabuilder-sw.js`  
```

Comment:
Can we delete the bulleted list since the links have now been moved up?

Reply:
sure